### PR TITLE
World Vision Clinic Application Development (Front-End)

### PR DESCRIPTION
**Pull request by MudasirSherwani**
### Our team consists of 4 members 🔢 
1. [Abdul Saboor Sohaib](https://github.com/SaboorSohaib)
2. [Mudasir Sherwani](https://github.com/MudasirSherwani)
3. [Olaoye Oluwatoyin Abayomi](https://github.com/AbayomiOlaoye)
4. [Zainab Al-Zubair](https://github.com/Zainab-Alzubair)

### [Live Demo URL](https://world-vision-clinic.netlify.app/)
**Default Username: mudasir@gmail.com Password: 123456**
- Follow the layout of [the provided design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
- The user logs in to the website, 
- In the navigation panel, the user can see links to:
    - Doctors 
    - "Appointment" form.
    - "My Appointments".
- "Add the Doctor that you selected as a theme" (the link is visible to everybody).
- "Delete the Doctor that you selected as a theme" (the link is visible to everybody).
- On the main page, the user can see a [list of Doctors that you selected as a theme](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005577).
- When the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579)
- When the user clicks the "Add Doctor" link in the navigation panel they can see a form for adding a new Doctor.
- Make the app responsive, creating both mobile and desktop versions.
- Fix Linters
- Follow GitFlow
- Add README
- Deployment has been done with both (Front-end & Back-end)

## Kanban Board:
image

### CODE REVIEW

File: src/redux/user/user.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+//         'Content-Type': 'application/json',
+//         Authorization: token,
+//       },
+//     });
+//     toast.success('Logged out successfully');
+//     return response.data;
+//   } catch (error) {
+//     throw new Error(error);
+//   }
+// };
```
Comment (Meltrust) [inline src/redux/user/user.js:45]: - [ ] Kindly, remove this piece of old commented code in order to improve the repo's cleanliness. 👍

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
+import React from 'react';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import { useSelector } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import AuthenticatedRoute from './routes/AuthenticatedRoute';
+import UnAuthenticatedRoute from './routes/UnAuthenticatedRoute';
 
 function App() {
```
Comment (Meltrust) [inline src/App.js:10]: - [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

image

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @MudasirSherwani,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Nice code organization ✔️ 
-  App is working well✔️ 
-  Good readme ✔️ 
-  Beautiful frontend ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: src/App.js (hunk 2 above)
Comment (MudasirSherwani) [reply src/App.js]: image

**Review by Meltrust — APPROVED** [review]
#### Hi @MudasirSherwani,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Commented code has been removed✔️ 
- Functions are now ES6✔️ 
- Beautiful frontend✔️
- Superb job ✔️  



Cheers and Happy coding!👏👏👏


------

**Pull request merged**